arctic/store/_ndarray_store.py

```diff
@@ -120,6 +121,28 @@ def set_corruption_check_on_append(enable):
     _CHECK_CORRUPTION_ON_APPEND = bool(enable)
 
 
+def update_fw_pointers(collection, symbol, version, previous_version, ids_of_updated_segments=None, result=None):
+    # If this is the first use of fw-pointers, query for the full list of segment IDs,
+    # or continue by extending the IDs from the previous version
+    if previous_version and FW_POINTERS_KEY not in previous_version:
+        segment_ids = {_id for _id in collection.find({'symbol': symbol, 'parent': version_base_or_id(version)},
```

Comment:
Worth asserting here the segment id count is right (as is done in the read path)?